skimage/data/tests/test_data.py

```diff
@@ -116,24 +107,8 @@ def test_cell():
     data.cell()
 
 
-@skipif(OFFLINE, reason='No internet connection')
 def test_cells_3d():
     """Needs internet connection."""
     path = fetch('data/cells.tif')
     image = io.imread(path)
     assert image.shape == (60, 256, 256)
-
-
-def test_data_dir():
```

Comment:
Why remove this test? I feel like it's important.

Reply:
Thanks, I added it back.